### PR TITLE
Make docs build examples strictly

### PR DESCRIPTION
**Pull request by ChrisRackauckas**

### CODE REVIEW

File: docs/Project.toml [hunk 1]
```diff
@@ -1,5 +1,11 @@
...
+CairoMakie = "13f3f980-e62b-5c42-98c6-ff1f3baf88f0"
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+GlobalSensitivity = "af5da776-676b-467e-8baf-acd8249e4f0f"
+OrdinaryDiffEq = "1dea7af3-3e70-54e6-95c3-0bf5283fa5ed"
+Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
+QuasiMonteCarlo = "8a4e6c94-4038-4cdc-81c3-7e6ffdb2a71b"
+Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
 [compat]
 Documenter = "0.25.3"
```
Comment (Vaibhavdixit02) [inline docs/Project.toml:11]:
```suggestion
Documenter = "0.27"
```

**Comment by ChrisRackauckas** [comment]
What was `samples` supposed to be here? https://github.com/SciML/GlobalSensitivity.jl/runs/7731725582?check_suite_focus=true#step:5:42

**Comment by ChrisRackauckas** [comment]
The builds are killing the memory on the machines.

File: docs/src/tutorials/juliacon21.md [hunk 2]
```diff
@@ -22,22 +22,23 @@ t = collect(range(0, stop=10, length=200))
 f1 = function (p)
     prob1 = remake(prob;p=p)
     sol = solve(prob1,Tsit5();saveat=t)
+    return [mean(sol[1,:]), maximum(sol[2,:])]
 end
 
 bounds = [[1,5],[1,5],[1,5],[1,5]]
 
-reg_sens = gsa(f1, RegressionGSA(true), bounds)
+reg_sens = gsa(f1, RegressionGSA(true), bounds, samples = 1000)
```
Comment (ChrisRackauckas) [inline docs/src/tutorials/juliacon21.md:30]:
```suggestion
reg_sens = gsa(f1, RegressionGSA(true), bounds, samples = 200)
```

File: docs/src/tutorials/juliacon21.md [hunk 3]
```diff
@@ -49,9 +50,9 @@ fig
 
 ![morrisscat](https://user-images.githubusercontent.com/23134958/127019346-2b5548c5-f4ec-4547-9f8f-af3e4b4c317c.png)
 
-```julia
+```@example lv
 sobol_sens = gsa(f1, Sobol(), bounds, samples=5000)
```
Comment (ChrisRackauckas) [inline docs/src/tutorials/juliacon21.md:54]:
```suggestion
sobol_sens = gsa(f1, Sobol(), bounds, samples=500)
```

File: docs/src/tutorials/juliacon21.md [hunk 4]
```diff
@@ -49,9 +50,9 @@ fig
...
-efast_sens = gsa(f1, eFAST(), bounds)
+efast_sens = gsa(f1, eFAST(), bounds, samples=5000)
```
Comment (ChrisRackauckas) [inline docs/src/tutorials/juliacon21.md:55]:
```suggestion
efast_sens = gsa(f1, eFAST(), bounds, samples=500)
```

File: docs/src/tutorials/juliacon21.md [hunk 5]
```diff
@@ -70,7 +71,7 @@ fig
 ![sobolefastprey](https://user-images.githubusercontent.com/23134958/127019361-8d625107-7f9c-44b5-a0dc-489bd512b7dc.png)
 ![sobolefastpred](https://user-images.githubusercontent.com/23134958/127019358-8bd0d918-e6fd-4929-96f1-d86330d91c69.png)
 
-```julia
+```@example lv
 using QuasiMonteCarlo
 samples = 5000
```
Comment (ChrisRackauckas) [inline docs/src/tutorials/juliacon21.md:76]:
```suggestion
samples = 500
```

File: docs/src/tutorials/parallelized_gsa.md [hunk 6]
```diff
@@ -72,7 +72,7 @@ Doing it in this manner lets us directly specify a quasi-Monte Carlo sampling me
 we use [QuasiMonteCarlo.jl](https://github.com/SciML/QuasiMonteCarlo.jl) to generate the design matrices
 as follows:
 
-```julia
+```@example designmat
 using GlobalSensitivity, QuasiMonteCarlo, Plots
 samples = 10000
```
Comment (ChrisRackauckas) [inline docs/src/tutorials/parallelized_gsa.md:77]:
```suggestion
samples = 500
```

**Pull request merged**